cmd/proxy/app/proxy.go

```diff
@@ -66,6 +66,11 @@ func NewProxyCommand() *cobra.Command {
 				if err != nil {
 					klog.Fatalf("Failed to create proxy server, %v", err)
 				}
+				watcher, err := p.WatchSVC()
```

Comment:
Why don't we use Informer with event handler, for example https://github.com/kubesphere/tower/blob/master/pkg/proxy/proxy.go#L112, that's more recommended by K8s.

Reply:
refacted